sdk/state/state.go

```diff
@@ -79,13 +81,21 @@ func NewChannelFromSnapshot(c Config, s Snapshot) *Channel {
 	return channel
 }
 
+// ChannelAccount is an account that, with another channel account, form a
+// channel. Two participants are needed to form a channel, referred to as remote
+// and local.
+//
+// The channel accounts hold the assets that are used for payments and will be
+// re-distributed at close.  The initiator channel account's sequence number
+// serves as the sequence number for the channel.
```

Comment:
Similar here, I wouldn't identify initiator because again it's not clear from looking at the fields within which field identifies the account as an initiator.

I think these details are awesome, but they belong in the package doc comment.

Reply:
👍 
https://github.com/stellar/starlight/pull/436/commits/16344c3b9e9566b1307dae144e7b7c1924f6869c